### ISSUE TITLE
check_ping - output error [sf#946857]

### ISSUE THREAD

**Issue by monitoring-user** [monitoring-plugins/monitoring-plugins#312]
Submitted by pumuckel1980 on 2004-05-03 12:05:17

Hello,

I have some problems with the "nagios-plugins-
1.4.0alpha1". In this Version the check_ping plugin show 
me the following result:
"bin/ping -n -U -c 1 ntest2ext
Error: Could not interpret output from ping command"

It's OK when I use the "nagios-plugins-1.3.1.tar.gz".

./check_ping -H ntest2ext -w 3000.0,80% -c 
5000.0,100% -p 1 PING OK - Packet loss = 0%, RTA = 
0.22 ms
PING OK - Packet loss = 0%, RTA = 0.46 ms

Here are my differnet settings:

1.
nagios-plugins-1.3.1.tar.gz
###### 

= uname -a
"Linux _HOSTNAME_ 2.4.22 #1 SMP Mit Okt 29 15:21:08 
CET 2003 i686 i686 i386 GNU/Linux"

=./check_ping -H _HOSTNAME_ -w 3000.0,80% -c 
5000.0,100% -p 1 
"PING OK - Packet loss = 0%, RTA = 0.22 ms
PING OK - Packet loss = 0%, RTA = 0.48 ms"
1. 
   "nagios-plugins-1.4.0alpha1"
   ##############################
   = uname -a
   "Linux _HOSTNAME_ 2.4.22 #1 SMP Mit Okt 29 1:21:08 
   CET 2003 i686 i686 i386 GNU/Linux"

=./check_ping -H ntest2ext -w 3000.0,80% -c 
5000.0,100% -p 1
"/bin/ping -n -U -c 1 ntest2ext
Error: Could not interpret output from ping command"

thanks for help
bye
